Cubical/Categories/Instances/Discrete.agda

```diff
@@ -0,0 +1,55 @@
+-- Discrete category over a type A
+-- A must be an h-groupoid for the homs to be sets
+{-# OPTIONS --safe #-}
+
+module Cubical.Categories.Instances.Discrete where
+
+open import Cubical.Categories.Category.Base
+open import Cubical.Categories.Functor.Base
+open import Cubical.Foundations.GroupoidLaws
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Transport
+
+private
+  variable
+    ℓ ℓC ℓC' : Level
+
+open Category
+
+-- Discrete category
+Discrete : hGroupoid ℓ → Category ℓ ℓ
```

Comment:
Rename to `DiscreteCategory` ?

Reply:
My suggestion would be more consistent with https://github.com/agda/cubical/blob/master/Cubical/Categories/Instances/Rings.agda. If someone wants a shorter name they can rename when importing.

Reply:
Okay, will do.